### PR TITLE
[merged]  Fix up many failures when running atomic with overlay backend

### PR DESCRIPTION
**Pull request by rhatdan**
Overlay fields are being returned from docker as Capatalized

LowerDir, UpperDir, WorkDir

### CODE REVIEW

**Comment by rhatdan** [comment]
@baude PTAL

File: Atomic/mount.py [hunk 1]
```diff
@@ -636,6 +643,9 @@ def _clean_temp_container_by_path(self, path):
             self.d.remove_container(short_cid)
         self._clean_tmp_image()
 
+    def get_driver(self):
```
Comment (baude) [inline Atomic/mount.py]: Is this function needed?

File: Atomic/mount.py [hunk 2]
```diff
@@ -617,7 +617,14 @@ def _get_overlay_mount_cid(self):
...
             raise MountError('The device mounted at that location is not a '
                              'docker container.')
-        return cdir.replace('/var/lib/docker/overlay/', '')
+
+        for c in self._get_all_cids():
+            graph = self.d.inspect_container(c)["GraphDriver"]
+            if graph['Data']['UpperDir'].startswith(cdir):
+                return c
+
+        raise MountError('The device mounted at that location is not a '
```
Comment (baude) [inline Atomic/mount.py]: Should "that location" be cdir or something specific?

File: Atomic/mount.py (hunk 1 above)
Comment (rhatdan) [inline Atomic/mount.py]: I was using it but figured an easier way.  But I thought it was a worth while function to add.

File: Atomic/mount.py (hunk 2 above)
Comment (rhatdan) [inline Atomic/mount.py]: Fixed the error message.

**Comment by baude** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**